frontend/src/app/applications-upload/applications-upload.page.ts

```diff
@@ -108,10 +102,10 @@ export class ApplicationsUploadPage implements OnInit {
        * picture of applicant and documents is mapped according json format required by server.
        */
       images.push({
-        content: inp.profilePicture.pictureBase64,
+        content: inp.profilePicture?.pictureBase64,
```

Comment:
I am not sure if this is how the server expects the data. Shouldn't everything here rather be wrapped in an if statement? Otherwise, the server will receive a profile picture with no content.

Reply:
actually the profile picture is not part of the skipped steps, so i just deleted the "?". So it works for the moment, because there will be always a profile picture atm